rbac/management/group/view.py

```diff
@@ -253,6 +253,10 @@ def update(self, request, *args, **kwargs):
                 "name": "GroupA"
             }
         """
+        group = Group.objects.filter(uuid=kwargs.get('uuid')).first()
```

Comment:
I am wondering should we modify the flag here? I thought we would only change it to False if we just change the roles. Here this update seems only updating the name or description?

Reply:
I went back and forth on that a bit, and my feeling was that we'd likely want ANY edit to a platform_default role, even just a name change, to modify the system flag. If we don't care so much in this case, we can pull this flag change out, though.

Reply:
@wcmitchell @astrozzc suggest confirming with Nicole. But I also suggest we flip the switch for any change to keep things simple.

Reply:
Based on the UX presentation last demo, I'm not sure the group name for the default groups will even be editable via the UI? It looks like they plan to auto-update the name when a role is added/removed, which are the triggers related to managing/not managing the group. So my only thought would be, that may be a disconnect between the UI and the API.

Maybe we actually want to prevent the names of platform default groups from being edited through the API, and have a hook during this process to update the name in a standard way for consistency, per the UX expectations? So, flip the flag to false and prepend the name in a standard way, but don't allow the platform default groups' names to be edited. Just a thought.

Reply:
OK👍 I  will keep in mind of the APIs UI have to add info for customers to maintain the group for them.